eZ/Publish/API/Repository/Tests/SearchServiceFulltextEmbedTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests;
```

Comment:
Please create `eZ\Publish\API\Repository\Tests\SearchService` namespace, it will be easier to maintain.

Reply:
Moved all search-service tests to a new namespace

Reply:
OK, as we discussed I've created a new namespace and add there only one new test